src/foundation.cs

```diff
@@ -4452,19 +4452,15 @@ public partial interface NSUrl : NSSecureCoding, NSCopying
 		[Export ("initFileURLWithPath:isDirectory:")]
 		IntPtr Constructor (string path, bool isDir);
 
+		[Internal]
 		[Export ("initWithString:")]
-		IntPtr Constructor (string urlString);
+		IntPtr _Init (string urlString);
 
+		[Internal]
```

Comment:
that's hiding the `[DesignatedInitializer]` ... so it hide the information (not very useful today) from people who might want to subclass (correctly) the type